packages/blocks/src/code-block/components/lang-list.ts

```diff
@@ -229,49 +234,86 @@ export class LangList extends LitElement {
     return this;
   }
 
+  protected updated() {
+    if (this.showLangList !== 'hidden') {
+      this.filterInput.focus();
+    }
+  }
+
+  protected firstUpdated() {
+    document.addEventListener('click', (e: MouseEvent) => {
```

Comment:
Can we limit the scope of this listener to the `editor-container`?

Reply:
If we need to detect the click outside code block language select container, add listener to document may be needed, but the handler will be removed after disposing, would this be viable?
https://github.com/toeverything/blocksuite/blob/d1b177466e5468a238f72ed0a1d38140af4b3f0f/packages/blocks/src/code-block/components/lang-list.ts#L258

I thought appending a fake dom of 100% width and height to `<body>`'s children, and add a listener to the fake dom. But since we can remove the handler of document, so I considered the current implementation finally)